language-design.md

```diff
@@ -17,6 +17,18 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+1. "I should not design a small language, and I should not design a large one. I need to design a language that can grow. I need to plan ways in which it might grow—but I need, too, to leave some choices so that other persons can make those choices at a later time" [Steele, 1998].
+
+   I found this perspective interesting, as it related to some of the in-class discussions we have had about addressing the needs of users and languages. We need to leave room for the users, even though an implementer may be inclined to prioritize the functionality they desire, they should prioritize their language being broadly usable over something strictly holding true to their original image.
+
+2. "They created a “placebo language” called Randomo, whose syntax was randomly generated, to use in trials alongside Quorum and Perl. Novice programmers were able to write sample programs more accurately in Quorum versus Perl" [Pavlus, 2012].
+   
+   I found this result to be very surprising because I would have expected an intentionally designed programming language to better suit the needs of its users than a "placebo language" with randomly generated syntax. I wonder how this was generated because many programming languages hold similar characterisitcs so if Randomo did not highly vary from the languages it was modeled after, then this result would make much more sense to me.
```

Comment:
Definitely not a great look for Perl… I wonder how experienced users feel about these results.